bot/lang_ru.py

```diff
@@ -17,9 +16,9 @@
 
 RU_COMMON_HANDLERS_BUTTONS: dict[str, list[str]] = {
     'roles': ["Backend", "Frontend", "Mobile", "DevOps", "QA", "Marketing", "PM", "PO", "CTO", "CEO", "CPO",
-              "UI Designer", "UX Designer", "ML", "Contentmaker", "TechLead", "HR", "GameDev", "Graphic Designer",
-              "Data Analytic"],
-    'choose_experience': ["До года", "1-3 года", "3-5 лет", "Свыше 5 лет", "Не работаю"],
+              "UI Designer", "UX Designer", "Graphic Designer", "3D Designer", "Motion Designer", "ML",
+              "Contentmaker", "TechLead", "HR", "GameDev", "Data Analytic"],
```

Comment:
Please, change "Data Analytic" to "Data Analyst" (this is a mistake that was not made by you, but since this block of code is being edited, I suggest fixing it here so that everything could be in one place)